client/landing/subscriptions/components/site-subscription-page/site-subscription-details.tsx

```diff
@@ -84,6 +44,18 @@ const SiteSubscriptionDetails = ( {
 		error: unsubscribeError,
 	} = SubscriptionManager.useSiteUnsubscribeMutation( blogId );
 
+	const confirmUnsubscribe = ( { blogId, url }: { blogId: string; url: string } ) => {
+		if (
+			confirm(
+				translate(
+					'You currently have paid subscriptions with this site. Paid subscriptions must be cancelled separately by clicking the Manage Subscriptions button or going to https://wordpress.com/me/purchases. Press OK to proceed with unsubscribing from the site. Press Cancel to go back.'
+				)
+			)
```

Comment:
Let's not translate this. And the `\n\n` is missing.

```suggestion
			confirm("You currently have paid subscriptions with this site. Paid subscriptions must be cancelled separately by clicking the Manage Subscriptions button or going to https://wordpress.com/me/purchases.\n\nPress OK to proceed with unsubscribing from the site.\nPress Cancel to go back.");
```